src/components/navigation/Navbar.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { UserAuth } from "../../context/AuthContext";
+import "./Navbar.css";
+
+function Navbar() {
+  const { user, logOut } = UserAuth();
+
+  const handleSignOut = async () => {
```

Comment:
What would the end state of this look like?
Would you import the utilities.js at the top and then call something like `HandleSignOut()`?

Reply:
That is correct. The main purpose of moving it to a file is to avoid writing duplicate code. The moment we copy/paste code and not change anything, that should be a red flag moving forward

Reply:
created a custom hook in utilities and used that inside the functional component {handleSignOut}. Seems to work